pkg/provider/aws/resources/vpc.go

```diff
@@ -68,26 +69,37 @@ type (
 	}
 )
 
-func CreateNetwork(appName string, dag *core.ResourceGraph) *Vpc {
-
+func CreateNetwork(config *config.Application, dag *core.ResourceGraph) *Vpc {
+	appName := config.AppName
 	vpc := NewVpc(appName)
 
 	if dag.GetResource(vpc.Id()) != nil {
 		return vpc
 	}
 
 	region := NewRegion()
+	azs := NewAvailabilityZones()
 	igw := NewInternetGateway(appName, "igw1", vpc)
 
 	dag.AddResource(region)
+	dag.AddResource(azs)
```

Comment:
nit: `AddDependency` will add the resources for you if they don't exist, so this isn't needed